docs: Updates .NET Agent documentation to include support  for automatic browser injection in ASP.NET Core 6+ web applications (#15051)
newrelic/docs-website: src/content/docs/apm/agents/net-agent/other-features/browser-monitoring-net-agent.mdx

## Auto-instrumentation [#auto_instrumentation]

<Callout variant="important">

>   Auto-instrumentation is only available for .NET Framework apps and .NET Core v6.0 and later web apps. Auto-instrumentation is unavailable for ASP.NET Core v5.0 and earlier applications whether they're monitored by the .NET Framework or Core agent.

Reviewer: Let's clarify the last sentence. ".NET Framework or Core agent" doesn't make sense because the .NET agent monitors both .NET framework and .NET core". Can you clarify what you mean here?

Author: There's some goofy naming inconsistency at play, thanks to Microsoft. 

.NET Framework applications can include ASP.NET Core web apps, using ASP.NET Core v3 or earlier. .NET Core applications can include ASP.NET Core web apps version 3 or later. We don't support browser instrumentation on ASP.NET Core web applications for versions less than 6.

Reviewer: Would this be correct then?
```suggestion
  Auto-instrumentation is only available for .NET Framework apps and .NET Core v6.0 and later web apps. Auto-instrumentation is unavailable for ASP.NET Core v5.0 and earlier applications even if they're monitored by the .NET agent."
```
